### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by chetna14manku**
[MSINTEROP-810](https://issues.redhat.com/browse/MSINTEROP-810): Update Readme

### CODE REVIEW

**Comment by msi-b1** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
    * You can add extra args to the Podman build command
        * Example: `/build-and-push-container --build-arg OPENSHIFT_PYTHON_WRAPPER_COMMIT=<commit_hash>`
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
 * To assign reviewers based on OWNERS file use `/assign-reviewers`
 * To check if PR can be merged use `/check-can-merge`

<details>
<summary>Supported /retest check runs</summary>

 * `/retest tox`: Retest tox
 * `/retest python-module-install`: Retest python-module-install

</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

File: README.md [hunk 1]
```diff
@@ -26,7 +31,24 @@ git pull
...
+```
+git clone https://github.com/RedHatQE/rosa-python-client.git
+cd rosa-python-client
+poetry install
+```
+
+From pypi:
+
+```bash
+pip install rosa-python-client --user
```
Comment (myakove) [inline README.md:47]: It's better to use `pipx` for this (pipx create venv for each application so your OS stay clean)

**Review by myakove — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chetna14manku) [reply README.md:47]: @myakove when I tried to install it using `pipx install rosa-python-client` getting this, I believe for this package pipx cannot be used
```
No apps associated with package rosa-python-client. Try again with '--include-deps' to include apps of dependent packages, which are listed above. If you
are attempting to install a library, pipx should not be used. Consider using pip or a similar tool instead.
```

**Review by myakove — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**